searx/webapp.py

```diff
@@ -429,7 +429,7 @@ def render(template_name: str, **kwargs):
     kwargs['theme'] = request.preferences.get_value('theme')
     kwargs['method'] = request.preferences.get_value('method')
     kwargs['categories_as_tabs'] = list(settings['categories_as_tabs'].keys())
-    kwargs['categories'] = _get_enable_categories(categories.keys())
+    kwargs['categories'] = _get_enable_categories(settings['categories_as_tabs'].keys())
```

Comment:
I'm missing something: I can't find the usage of `categories` in the template.

Reply:
`categories_as_tabs` is used in the preferences.html and `categories` is used in the categories.html (see above in this patch).